packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -302,7 +303,10 @@ export default function MainFeedLayout({
       return;
     }
 
-    openModal({ type: LazyModal.MigrateUserStreak });
+    openModal({
```

Comment:
This one is not a experiment?

Reply:
It is, but the condition above considers the check for that. Should be safe.

Reply:
Why do we have it on Feed and MainFeedLayout as well?

Reply:
They are two different promotions. The one on Feed is the Bookmark loops, this one is `promotion.migrateStreaks`.